xapian-core/Makefile.am

```diff
@@ -105,8 +105,7 @@ lib_LTLIBRARIES = libxapian-1.5.la
 libxapian_1_5_la_SOURCES = $(lib_src)
 libxapian_1_5_la_LIBADD = $(XAPIAN_LIBS)
 libxapian_1_5_la_LDFLAGS = \
-    $(XAPIAN_LDFLAGS) $(NO_UNDEFINED) -version-info $(LIBRARY_VERSION_INFO) $(lib_ldflags)
-
+    $(XAPIAN_LDFLAGS) $(NO_UNDEFINED) -version-info $(LIBRARY_VERSION_INFO) $(lib_ldflags) $(ICU_LIBS)
```

Comment:
Unless `ICU_LIBS` is misleadingly named, it ought to go in `LIBADD` not `LDFLAGS`.